packages/babel-preset/common.js

```diff
@@ -0,0 +1,100 @@
+const nonStandardPlugins = require('./non-standard-plugins');
+
+module.exports = function shopifyCommonPreset(api, options = {}) {
+  const {
+    targets = {
+      node: 'current',
+    },
+    corejs = 3,
+    debug = false,
+    modules = 'auto',
+    typescript = false,
+    useBuiltIns = 'entry',
+    transformRuntime = false,
+    transformRuntimeOptions = {
+      corejs: false,
+      helpers: true,
+      // By default, babel assumes babel/runtime version 7.0.0-beta.0,
+      // explicitly resolving to match the provided helper functions.
+      // https://github.com/babel/babel/issues/10261
+      version: require('@babel/runtime/package.json').version,
+      regenerator: true,
+      // https://babeljs.io/docs/en/babel-plugin-transform-runtime#useesmodules
+      // We should turn this on once the lowest version of Node LTS
+      // supports ES Modules.
+      useESModules: false,
+      // This allows users to run transform-runtime broadly across a whole project.
+      // By default, transform-runtime imports from @babel/runtime/foo directly, but
+      // that only works if @babel/runtime is in the node_modules of the file that is being compiled.
+      absoluteRuntime: false,
+    },
+  } = options;
+
+  const isNode = 'node' in targets;
+
+  const presets = [
+    [
+      require.resolve('@babel/preset-env'),
+      {
+        modules,
+        useBuiltIns,
+        corejs,
+        targets,
+        debug,
+        bugfixes: true,
+      },
+    ],
+  ];
+
+  const plugins = [
+    ...nonStandardPlugins(options),
```

Comment:
this seems to duplicate most of the inline definitions you've provided here. Can it be removed?

Reply:
Thanks. I missed it, this was my intention by moving its contents into here.